model/config.go

```diff
@@ -3129,6 +3129,52 @@ type Config struct {
 	ExportSettings            ExportSettings
 }
 
+func (o *Config) Auditable() map[string]interface{} {
```

Comment:
I think we may want to return an empty interface for this one until we have it properly implemented. As you noted in the API handler it may make sense to just print the diff for the config in the Prior and Result state, but that would use a different mechanism. The risk here is if someone would use it they would also log all sensitive secrets.